optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -68,8 +69,12 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
         StringBuilder standardDeviationString = new StringBuilder(standardDeviationDoubles.length * 9);
         // Abbreviate to 2 decimals
         // We don't use a local sensitive DecimalFormat, because other Scores don't use it either (see PLANNER-169)
+        DecimalFormatSymbols decimalFormatSymbols = DecimalFormatSymbols.getInstance();
```

Comment:
Definitely, thanks for catching that.

I'd prefer the following, as it doesn't depend on any *specific* locale being present (uses the default one):
```
DecimalFormatSymbols decimalFormatSymbols = new DecimalFormatSymbols();
decimalFormatSymbols.setDecimalSeparator('.');
DecimalFormat exponentialFormat = new DecimalFormat("0.0#E0", decimalFormatSymbols);
DecimalFormat decimalFormat = new DecimalFormat("0.0#", decimalFormatSymbols);
```

WDYT?